odata-json-format/7 Structural Property.md

```diff
@@ -138,8 +138,8 @@ Example ##ex: partial collection of strings with next link
 ```
 :::
 
-A collection of primitive values that occurs in a property of type `Edm.Untyped`
-is interpreted as a collection of `Edm.Boolean`, `Edm.String`, and `Edm.Decimal` values,
+A primitive value within an untyped collection of primitive values
```

Comment:
```suggestion
A primitive value within an untyped collection
```
Can untyped collections contain a mix of primitive, object, and array values?

Reply:
Why have this here?

Better move it to section "Untyped Value" and link to this section here.

Reply:
Moved.